MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,133 @@
+You may enable normalization of custom keys by setting the option `sdk.service.operation({ normalizeCustomData: true })`
+- Before 8.0.0: `sdk.profiles.update(id, data, options)`
+- After 8.0.0: `sdk.profiles.update(id, data, {...options, normalizeCustomData: true ))`
+
+### Reverting to request normalization of custom keys for all operation a client
+The normalization of custom keys can be reverted to behavior before 8.0.0 using the snippet `sdk = createClient({ ...options, normalizeCustomData: true });`.
+- You may then disable normalization of custom keys for a single operation by  `sdk.service.operation({ normalizeCustomData: false })`
+
+### Affected Properties
+
+- `sdk.configurations.find`
+  - Response: `data`
+  - Response: `userConfiguration`
+  - Response: `groupConfiguration`
+  - Response: `staffConfiguration`
+  - Response: `patientConfiguration`
+- `sdk.configuration.groups.find`
+  - Response: `data`
+  - Response: `staff`
+  - Response: `patientConfiguration`
+- `sdk.configurations.users.find`
+  - Response: `data`
+  - Response: `staffConfigurations.data`
+  - Response: `patientConfigurations.data`
+- `sdk.dispatchers.actions.create`
+  - Response: `data`
+- `sdk.dispatchers.get`
+  - Response: `data.actions.data`
+- `sdk.dispatchers.create`
+  - Response: `action.data`
+- `sdk.localizations.getByKeys`
+  - Response: `ALL KEYS`
+- `sdk.mails.find`
+  - Response: `data.content`
+- `sdk.mails.send`
+  - Request & Response: `content`
+- `sdk.mails.findOutBound`
+  - Response: `data.templateData.content`
+- `sdk.payments.orders.find`
+  - Response: `data.data`
+  - Response `data.product.schema.properties`
+- `sdk.payments.orders.create`
+  - Response: `data`, 
+  - Response: `product.schema.properties`
+- `sdk.payments.products.create`
+  - Response `schema.properties`
+- `sdk.payments.products.find`
+  - Response `data.schema.properties'`
+- `sdk.profiles.groups.create`
+  - Request & Response: `customFields`
+- `sdk.profiles.groups.update`
+  - Request & Response: `customFields`
+- `sdk.profiles.groups.removeFields`
+  - Response: `customFields`
+- `sdk.profiles.create`
+    - Request & Response: `customFields`
+    - Request & Response: `groups.customFields`
+- `sdk.profiles.find`
+    - Request & Response: `data.customFields`
+    - Request & Response: `data.groups.customFields`
+- `sdk.profiles.update`
+    - Request: `customFields`
+    - Request: `groups.customFields`
+- `sdk.task.find`
+  - Response: `data.data`
+- `sdk.tasks.create`
+  - Response: `data`
+- `sdk.tasks.api.*`
+    - Response: `ALL`
+- `sdk.tasks.functions`
+  - Response: `data`
+- `sdk.tasks.schedules.find`
+  - Response: `data.data`
+- `sdk.tasks.schedules.create`
+  - Response: `data`
+- `sdk.templates.find`
+  - Response: `schema.fields`
+  - Response: `fields`
+- `sdk.templates.create`
+  - Response: `schema.fields`
+  - Response: `fields`
+- `sdk.templates.resolveAsJson`
+  - Response: `*`
```

Comment:
(minor) I like the `All keys` or `ALL` you've used above, might be nice to indicate that consistently